### DISCUSSION TITLE
Trying to find the file which needs to exist so firstrun.py doesn't trigger

### DISCUSSION THREAD

**Discussion by RenNagasaki** [erew123/alltalk_tts#452] [Dev Discussions]
Hey @erew123 I'm currently trying to identify which file script.py looks for to exist so it doesn't call firstrun.py.
I know it's called AlltalkConfig in the code but I'm kinda bad at python and fail to find out what the physical file is called.
script.py -> Line 835

Could you tell me? 😄 

Kind regards,
Ren

**Comment by erew123** [comment]
@RenNagasaki firstrun.py will always be called, but will do nothing if either one of two conditions is met, either:

Confignew.json has `"firstrun_model": true,`set to `false`, meaning a first time setup has been performed already.
https://github.com/erew123/alltalk_tts/wiki/Configuration-File:-confignew.json

Or the other option is:

script.py is being called with `--tts_model none` which is the same as selecting the none option on the first time run setup menu and will flag the above to false anyway.
https://github.com/erew123/alltalk_tts/blob/alltalkbeta/script.py#L39

**Reply by RenNagasaki** [reply]
Thanks!
